src/SCREENS.ts

```diff
@@ -22,6 +22,23 @@ const SCREENS = {
         WORKSPACES: 'Settings_Workspaces',
         SECURITY: 'Settings_Security',
         STATUS: 'Settings_Status',
+        PROFILE: 'Settings_Profile',
+        PRONOUNS: 'Settings_Pronouns',
+        DISPLAY_NAME: 'Settings_Display_Name',
+        TIMEZONE: 'Settings_Timezone',
+        TIMEZONE_SELECT: 'Settings_Timezone_Select',
+        PERSONAL_DETAILS_INITIAL: 'Settings_PersonalDetails_Initial',
```

Comment:
Can we make `PERSONAL_DETAILS` an object so that the hierarchy is clear? Just like we do for `SETTINGS`?